packages/example-getting-started/README.md

```diff
@@ -2,6 +2,9 @@
 
 This is the basic tutorial for getting started with Loopback 4!
 
+To get started, jump into the
+[Prerequisites and setup](docs/1-prerequisites-and-setup.html) section.
```

Comment:
Hmm, I think the link should be `docs/1-prerequisites-and-setup.md`. When it's imported into `loopback.io`, we probably need to enhance the script to change from `.md` to `.html`.

Reply:
I also prefer to list all sub tutorials here. I understand `1-prerequisites-and-setup.md` has navigation to other ones but having all of them visible at top README level allows users to jump.